pages/resources/quizzes/FirstLang/FirstProgrammingLanguageQuiz.tsx

```diff
@@ -63,7 +63,10 @@ const FirstProgrammingLanguageQuiz = () => {
       ...question,
       options: question.options.map((questionOption) =>
         questionOption.name === option.name
-          ? { ...questionOption, isSelected: true }
+          ? {
+              ...questionOption,
```

Comment:
clean code! good use of spread operator (as usual) and the ternary operator as well